db/migrate/20191205200030_create_recipes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRecipes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :recipes do |t|
+      t.string :title
```

Comment:
I'm surprised there is no constraint on `title`. Does the app handle a recipe with a `nil` title? Do you want it to?